chef_master/source/supermarket_backup_restore.rst

```diff
@@ -3,21 +3,86 @@ Supermarket Backup and Restore
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/supermarket_backup_restore.rst>`__
 
-Periodic backups of Supermarket data are an essential part of managing and maintaining a healthy configuration and to help ensure that important data can be restored, if required.
+Periodic backups of Supermarket data are an essential part of managing and maintaining a healthy configuration, and to help ensure that important data can be restored if required. In a typical installation of Supermarket, both the cookbook store and the database need to be backed up on a regular basis.
 
-**Cookbooks**
 
-The default location of cookbooks is here: `/var/opt/supermarket/data`
+Backup
+=====================================================
+
+Cookbook Backup
+-----------------------------------------------------
+
+If you are not using AWS S3 storage for cookbooks, you will also need to make a backup of the local cookbook storage location. 
```

Comment:
I'm thinking that the cookbook backup section should come after the database backup section since it would only apply if they are not using S3.  

Also I think the wording here should be:

If Supermarket is not configured to use AWS S3 storage for cookbooks, then the local cookbook storage location on the Supermarket server will also need to be backed up.